portal-web/docroot/html/taglib/ui/form_navigator/lexicon/sections.jspf

```diff
@@ -113,27 +113,26 @@
 				</c:otherwise>
 			</c:choose>
 
-			Liferay.once(
-				'<portlet:namespace />formReady',
-				function(event) {
-					if (!sectionContent.hasClass('in')) {
-						if (focusField.length) {
+			if (focusField.length) {
```

Comment:
Hey @jonmak08, is there any chance that we'll have a section that we'll need to expand that won't fulfill this requirement?  I switched things around to make sure that if nothing else, the panel would open (but I couldn't find a way to avoid doing that check twice).